client/src/components/App/index.js

```diff
@@ -1,11 +1,12 @@
 import React, { Component } from 'react';
+import Nav from '../Nav';
 import './App.css';
 
 class App extends Component {
   render() {
     return (
       <div>
-          Josh
+          <Nav />
```

Comment:
Yeah, quick run-through of what's going on here.

1. In components/Nav/index.js, we define a component. This component has a `render` method, which describes how that component should display itself (pretty much html)
2. We want to use the `Nav` component inside this component (think nested components) so we import it (`import Nav from '../Nav';`). Because the Nav component is defined in a file called index.js, node imports allow you to import the containing folder and it will automatically import the index.js file. Suppose the Nav component was defined in components/Nav/Nav.js (instead of components/Nav/index.js), then we would need `import Nav from '../Nav/Nav`. index.js is treated special.
3. When we use the component `<Nav />`, it will call the component's render function to render it here.